test-cases/longevity/longevity-5000-tables.yaml

```
@@ -29,6 +29,8 @@ regions_data:
     ami_id_loader: 'ami-0b94f0e897d884b1b'
   eu-west-1:
     ami_id_loader: 'ami-0bf19545bc7bc9e1f'
+  eu-north-1:
+    ami_id_loader: 'ami-036150abbde18c61a'
```

AFAIR and now Amos approved it, this loader AMI is the older version of the loader that has Java 8 and proved to work for this longevity.

You need to find the same version of loader also for the other regions and update them as well.

also, next to them add a comment with the reason we use this loader.
e.g.:
```suggestion
    ami_id_loader: 'ami-036150abbde18c61a'    # Loader AMI v5 (I made this number up) - with java 8 - last loader works for this test.
```
```suggestion
    ami_id_loader: 'ami-036150abbde18c61a'
```

---

OK, so the plan is covering all 6 supported regions for this loader AMI.
i think in this case we also have to add it similarly to the multi-keyspace yaml.